.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+on: push
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby_version:
+          - 3.1
+          - 3.2
+    steps:
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y -q libsqlite3-dev
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby_version }}
+          bundler-cache: true
+      - name: Install dependencies
+        run: bundle install
+      - name: Run Tests
+        run: bundle exec rspec
+
+  release-please:
+    name: Release Please
+    needs: [test]
```

Comment:
I know I have been back and forth on this in the past, but I believe release-please should not depend on tests to run. Mostly cause failing tests would prevent the Release PR from being updated, meaning there are changes in `main` which could be left out of the changelog if someone merged the Release PR while `main` is failing.

The `release` job which actually publishes the gem still depends on `test` though, so a new version won't actually be published in the event of failing tests. It'll only get a GitHub Release page on the repo without a corresponding gem to install.

Reply:
Cheers for explaining the latest thinking on the release please job not needing tests to pass, I've pushed up that change.